src/linker/ref/Linker/AssemblyAction.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Mono.Linker {
+
+	public enum AssemblyAction {
```

Comment:
Why do we need this? I don't see the `IStep` and related classes use this (or at least yet).

Reply:
I think that's leftover from the clean up